devicetypes/smartthings/zwave-thermostat.src/zwave-thermostat.groovy

```diff
@@ -282,7 +283,6 @@ def zwaveEvent(physicalgraph.zwave.commands.thermostatmodev2.ThermostatModeSuppo
 	if(cmd.heat) { supportedModes << "heat" }
 	if(cmd.cool) { supportedModes << "cool" }
 	if(cmd.auto) { supportedModes << "auto" }
-	if(cmd.auxiliaryemergencyHeat) { supportedModes << "emergency heat" }
```

Comment:
just comment this out please, in case at some point in the future we need to actually take this into account